google/cloud/security/scanner/scanners/instance_network_interface_scanner.py

```diff
@@ -0,0 +1,122 @@
+    """Pipeline to network enforcer from DAO."""
+
+    def __init__(self, global_configs, snapshot_timestamp):
+        """Initialization.
+
+        Args:
+            snapshot_timestamp: The snapshot timestamp
+        """
+        super(InstanceNetworkInterfaceScanner, self).__init__(
+            global_configs,
+            snapshot_timestamp)
+        self.snapshot_timestamp = snapshot_timestamp
+
+    def get_instance_networks_interfaces(self):
+        """Get network info from a particular snapshot.
+
+           Returns:
+               A list of networks from a particular project
+           Raises:
+               MySQLError if a MySQL error occurs.
+        I set the list  to get rid of duplicates
+        """
+        instances = instance_dao.InstanceDao().get_instances(
+            self.snapshot_timestamp)
+        network_interfaces = []
+        for instance in instances:
+            network_interfaces += instance.create_network_interfaces()
+        return network_interfaces
+
+    def parse_instance_network_instance(self, instance_object):
+        """Create a list of network interface obj."""
+        return instance_object.create_network_interfaces()
+
+    def _get_project_policies(self):
+        """Get projects from data source."""
+        project_policies = {}
+        project_policies = (
+            project_dao.ProjectDao().get_project_policies('projects',
+                                                          self.
+                                                          snapshot_timestamp))
+        return project_policies
+
+    @staticmethod
+    def _get_resource_count(project_policies, instance_network_interfaces):
+        """Get resource count for org and project policies.
+
+        Args:
+            org_policies: organization policies from inventory.
+            project_policies: project policies from inventory.
+        Returns:
+            Resource count map
+        """
+        resource_counts = {
+            ResourceType.PROJECT: len(project_policies),
+            ResourceType.INSTANCE: len(instance_network_interfaces),
+        }
+
+        return resource_counts
+
+    def run(self):
+        """Run the data collection."""
+        enforced_networks_data = []
+        project_policies = {}
+        instance_network_interfaces = self.get_instance_networks_interfaces()
+        enforced_networks_data.append(instance_network_interfaces)
+        enforced_networks_data.append(project_policies)
+
+        resource_counts = self._get_resource_count(project_policies,
+                                                   instance_network_interfaces)
+
+        return enforced_networks_data, resource_counts
+
+    def find_violations(self, enforced_networks_data, rules_engine):
+        """Find violations in the policies.
+
+        Args:
+        #TODO: should enforced_network_data be a gce_instance
+            enforced_networks_data: enforced networks data
```

Comment:
please indent the following line. Also, could you capitalize the first word in the arg description?